src/Features/CSharp/Portable/Completion/CompletionProviders/SymbolCompletionProvider.cs

```diff
@@ -74,6 +142,35 @@ protected override async Task<bool> IsSemanticTriggerCharacterAsync(Document doc
             return token.Kind() != SyntaxKind.NumericLiteralToken;
         }
 
+        /// <returns><see langword="null"/> if not an argument list character, otherwise whether the trigger is in an argument list.</returns>
+        private async Task<bool?> IsTriggerInArgumentListAsync(Document document, int characterPosition, CancellationToken cancellationToken)
+        {
+            var text = await document.GetTextAsync(cancellationToken).ConfigureAwait(false);
+            if (!CompletionUtilities.IsArgumentListCharacter(text[characterPosition]))
+            {
+                return null;
+            }
+
+            var tree = await document.GetSyntaxRootAsync(cancellationToken).ConfigureAwait(false);
+            var token = tree.FindToken(characterPosition);
+
+            if (!token.Parent.IsKind(SyntaxKind.ArgumentList, SyntaxKind.BracketedArgumentList, SyntaxKind.AttributeArgumentList, SyntaxKind.ArrayRankSpecifier))
+            {
+                return false;
+            }
+
+            // Only allow spaces between the end of the token and the trigger character
+            for (var i = token.Span.End; i < characterPosition; i++)
```

Comment:
we also need to validate that characterPositoin is *after* the token's end.

Reply:
Done.